assets/styles/components/_talk-to-us-text-box.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-talk-to-us{
+    padding:18px; 
+    background:#f8f8f8cc; 

```

Comment:
I don't think this colour will work as it's not a valid hex code?

Reply:
Actually ignore me, the last two characters are for alpha! Not something I've seen often (I usually use `rgba` for something like this, but I think that's purely personal preference)